src/auth/screens/login.screen.js

```diff
@@ -353,11 +353,7 @@ class Login extends Component {
               <View style={styles.browserSection}>
                 <WebView
                   source={{
-                    uri: `https://github.com/login/oauth/authorize?response_type=token&client_id=${
-                      CLIENT_ID
-                    }&redirect_uri=gitpoint://welcome&scope=user%20repo&state=${
-                      stateRandom
-                    }`,
+                    uri: `https://github.com/login/oauth/authorize?response_type=token&client_id=${CLIENT_ID}&redirect_uri=gitpoint://welcome&scope=user%20repo&state=${stateRandom}`,
```

Comment:
We need to fix the code style of this line, but probably not in this PR. For now, take out this change.